userguide/system.rst

```diff
@@ -147,54 +147,55 @@ can be configured using the General tab:
    |                      |                |                                                                                                                          |
    +----------------------+----------------+--------------------------------------------------------------------------------------------------------------------------+
-   | WebGUI IPv4 Address  | drop-down menu | choose from a list of recent IP addresses to limit the one to use when accessing the administrative GUI; the             |
-   |                      |                | built-in HTTP server will automatically bind to the wildcard address of *0.0.0.0* (any address) and will issue an        |
-   |                      |                | alert if the specified address becomes unavailable                                                                       |
+   | WebGUI IPv4 Address  | drop-down menu | Choose a recent IP address to limit the usage when accessing the administrative GUI. The                                 |
+   |                      |                | built-in HTTP server binds to the wildcard address of *0.0.0.0* (any address) and issues an                              |
+   |                      |                | alert if the specified address becomes unavailable.                                                                      |
    |                      |                |                                                                                                                          |
    +----------------------+----------------+--------------------------------------------------------------------------------------------------------------------------+
-   | WebGUI IPv6 Address  | drop-down menu | choose from a list of recent IPv6 addresses to limit the one to use when accessing the administrative GUI; the           |
-   |                      |                | built-in HTTP server will automatically bind to any address and will issue an alert                                      |
-   |                      |                | if the specified address becomes unavailable                                                                             |
+   | WebGUI IPv6 Address  | drop-down menu | Choose a recent IPv6 address to limit the usage when accessing the administrative GUI. The                               |
+   |                      |                | built-in HTTP server binds to any address issues an alert                                                                |
+   |                      |                | if the specified address becomes unavailable.                                                                            |
    |                      |                |                                                                                                                          |
```

Comment:
"if the specified address becomes unavailable." is on a new line and doesn't need to be

Reply:
will fix this.